<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDialog</class>
 <widget class="QDialog" name="SearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search Files</string>
  </property>
  <property name="windowIcon">
   <iconset theme="system-search">
    <normaloff/>
   </iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Name/Location</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>File Name Patterns:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLineEdit" name="namePatterns">
            <property name="text">
             <string>*</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="nameCaseInsensitive">
            <property name="text">
             <string>Case insensitive</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="nameRegExp">
            <property name="text">
             <string>Use regular expression</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Places to Search:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QListWidget" name="listView"/>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QPushButton" name="addPath">
                <property name="text">
                 <string>&amp;Add</string>
                </property>
                <property name="icon">
                 <iconset theme="list-add">
                  <normaloff/>
                 </iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="removePath">
                <property name="text">
                 <string>&amp;Remove</string>
                </property>
                <property name="icon">
                 <iconset theme="list-remove">
                  <normaloff/>
                 </iconset>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="recursiveSearch">
            <property name="text">
             <string>Search in sub directories</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="searchHidden">
            <property name="text">
             <string>Search for hidden files</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>File Type</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Only search for files of following types:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QCheckBox" name="searchTextFiles">
            <property name="text">
             <string>Text files</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="searchImages">
            <property name="text">
             <string>Image files</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="searchAudio">
            <property name="text">
             <string>Audio files</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="searchVideo">
            <property name="text">
             <string>Video files</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="searchDocuments">
            <property name="text">
             <string>Documents</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="searchFolders">
            <property name="text">
             <string>Folders</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Content</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>File contains:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLineEdit" name="contentPattern"/>
          </item>
          <item>
           <widget class="QCheckBox" name="contentCaseInsensitive">
            <property name="text">
             <string>Case insensiti&amp;ve</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="contentRegExp">
            <property name="text">
             <string>&amp;Use regular expression</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>186</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Properties</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>File Size:</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="0">
             <widget class="QCheckBox" name="smallerThan">
              <property name="text">
               <string>Smaller than:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QSpinBox" name="maxSize">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QSpinBox" name="minSize">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="largerThan">
              <property name="text">
               <string>Larger than:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QComboBox" name="minSizeUnit">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="currentIndex">
               <number>2</number>
              </property>
              <item>
               <property name="text">
                <string>Bytes</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>KiB</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>MiB</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>GiB</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QComboBox" name="maxSizeUnit">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="currentIndex">
               <number>2</number>
              </property>
              <item>
               <property name="text">
                <string>Bytes</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>KiB</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>MiB</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>GiB</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="title">
          <string>Last Modified Time:</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout">
            <item row="2" column="0">
             <widget class="QCheckBox" name="earlierThan">
              <property name="text">
               <string>Earlier than:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QCheckBox" name="laterThan">
              <property name="text">
               <string>Later than:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDateEdit" name="minTime">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDateEdit" name="maxTime">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SearchDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SearchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>largerThan</sender>
   <signal>toggled(bool)</signal>
   <receiver>minSizeUnit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>smallerThan</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxSize</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>largerThan</sender>
   <signal>toggled(bool)</signal>
   <receiver>minSize</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>smallerThan</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxSizeUnit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>laterThan</sender>
   <signal>toggled(bool)</signal>
   <receiver>minTime</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>earlierThan</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxTime</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
